internal/rest/client.go

```diff
@@ -379,14 +374,27 @@ func (c *Client) Close() {
 }
 
 // NewClient - returns new REST client.
-func NewClient(url *url.URL, tr http.RoundTripper, newAuthToken func(aud string) string) *Client {
+func NewClient(uu *url.URL, tr http.RoundTripper, newAuthToken func(aud string) string) *Client {
+	connected := int32(online)
```

Comment:
casting doesn't seem to be needed, LGTM otherwise

Reply:
It is - `online` is untyped, so unless specified it will be an `int`.